setup.py

```diff
@@ -24,25 +21,6 @@ def load_version():
     return globals_dict
 
 
-def is_installing():
```

Comment:
that's for another PR but as further clean up we could see if we could put the version in a text file in the package data and avoid having to execute version.py in the setup which is very messy